spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/serializer/ModuleSerializer.scala

```diff
@@ -131,7 +132,8 @@ object ModuleSerializer extends ModuleSerializable{
       // to make it compatible with both 2.11 and 2.10
       val ctorCs = clsSymbol.toType.declaration(universe.nme.CONSTRUCTOR)
       val primary: Option[universe.MethodSymbol] = ctorCs.asTerm.alternatives.collectFirst {
-        case cstor: universe.MethodSymbol if cstor.isPrimaryConstructor => cstor
+        case cstor if cstor.asInstanceOf[universe.MethodSymbol].isPrimaryConstructor =>
```

Comment:
@wzhongyuan Please take a look at this change here. The ```universe.MethodSymbol``` will be erased runtime.

Reply:
looks good